src/03-error-handling-async/index.test.ts

```diff
@@ -1,30 +1,57 @@
 // Uncomment the code below and write your tests
-// import { throwError, throwCustomError, resolveValue, MyAwesomeError, rejectCustomError } from './index';
+import {
+  throwError,
+  throwCustomError,
+  resolveValue,
+  MyAwesomeError,
+  rejectCustomError,
+} from './index';
 
 describe('resolveValue', () => {
   test('should resolve provided value', async () => {
-    // Write your test here
+    expect.assertions(1);
+    await expect(resolveValue('TEST')).resolves.toBe('TEST');
   });
 });
 
 describe('throwError', () => {
   test('should throw error with provided message', () => {
-    // Write your test here
+    expect.assertions(1);
+    const msg = 'test error';
+    try {
+      throwError(msg);
+    } catch (error) {
+      if (error instanceof Error) expect(error.message).toBe(msg);
+    }
   });
 
   test('should throw error with default message if message is not provided', () => {
-    // Write your test here
+    expect.assertions(1);
+    const defaultMsg = 'Oops!';
+    try {
+      throwError();
+    } catch (error) {
+      if (error instanceof Error) expect(error.message).toBe(defaultMsg);
+    }
```

Comment:
You should not catch error in the `catch`, but use mechanisms provided in jest.
For example, 
```
expect(() => {
      throwError();
    }).toThrow(defaultMessage);
```
Deducting 3 points.